client/src/components/Sidebar.js

```diff
@@ -1,90 +1,79 @@
+import _ from 'lodash';
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-
-import * as sdk from '../js-sdk/sdk'
-import SideZone from './SideZone.js';
-
-import '../styles/Sidebar.css';
-
 import {
   Container,
   Row,
-  Col
+  Col,
 } from 'reactstrap';
 
-  const exileList = [
-    "https://api.scryfall.com/cards/5d9aa740-9adf-412a-b6ec-0b9bb1b4618b",
-    "https://api.scryfall.com/cards/a4ee56f1-9f08-459f-8517-0fe7bc645fa3",
-    "https://api.scryfall.com/cards/2b7472f4-37b0-439f-b4ac-80706d40d191"
-  ];
-  const graveyardList = [ 
-    "https://api.scryfall.com/cards/fcdbb062-0b0b-4b4c-b4db-dd149f744baa",
-    "https://api.scryfall.com/cards/73db9c10-109a-417f-81cd-489d9510cc78",
-    "https://api.scryfall.com/cards/f40284e6-01a1-4372-a92c-940e5732607e"
-  ];
+import SideZone from './SideZone';
+import { Zones } from '../constants';
 
-class Sidebar extends Component {
-  constructor(props) {
-    super(props);
-  }
+import '../styles/Sidebar.css';
 
+class Sidebar extends Component {
   render() {
+    const { cards } = this.props;
+    const exile = _.filter(cards, (card) => _.get(card, 'state.zone') === Zones.EXILE);
+    const graveyard = _.filter(cards, (card) => _.get(card, 'state.zone') === Zones.GRAVEYARD);
```

Comment:
You could also use the new querying in the SDK for this rather than having the UI do it, and just taking in the two separate lists as props.

Reply:
I think for this project, as much as possible, it would be best to keep interactions with the SDK/listeners at the Container level (`GameContainer`) but yeah, the container could get all cards then sort or make a few different calls. I'm not super opinionated either way.